### ISSUE TITLE
[DOC] Unclear type of `MPI4PY_RC_IRECV_BUFSZ`

### ISSUE THREAD

**Issue by titodalcanton** [mpi4py/mpi4py#580]
Hi, the [documentation](https://mpi4py.readthedocs.io/en/stable/mpi4py.html#envvar-MPI4PY_RC_IRECV_BUFSZ) for the env var `MPI4PY_RC_IRECV_BUFSZ` reports the type to be `bool` and the default to be `True`, but also states that the value represents the "Default buffer size in bytes". Are the correct type and default value those given in https://mpi4py.readthedocs.io/en/stable/mpi4py.html#mpi4py.mpi4py.rc.irecv_bufsz?

**Comment by dalcinl** [comment]
Indeed, this is a documentation bug. The type is `int` and the default value is `32768`. Fixed in 26df94b86f2d4db0d5aee5810f2f01858a3aba84. Many thanks for the reporting the issue.
